docs/advanced/application-gateway/README.md

```diff
@@ -1,18 +1,24 @@
-# Integrating Application Gateway with Cloud Foundry on Azure
+# Integrating Application Gateway with CF on Azure
 
-To enable advanced feature like SSL termination on your Cloud Foundry network, you can integrate Azure Application gateway. This document provides steps to create and configure [Azure Application Gateway](https://azure.microsoft.com/en-us/services/application-gateway/), that will be used as load balancer with SSL offloading enabled for Cloud Foundry. We also introduce how to create the application gateway with a single certificate or multiple certificates.
+This document provides steps to create and configure [Azure Application Gateway](https://azure.microsoft.com/en-us/services/application-gateway/), and integrate it as load balancer with SSL offloading enabled for Cloud Foundry. This can be configured with either a single or multiple certificates.
 
-# 1 What is Azure Application Gateway
+## Azure Application Gateway background
+
+Azure Application Gateway provides application-level routing and load balancing services for your web front end, by replacing HaProxy with Azure Application Gateway, Cloud Foundry cusotmers gained engerprise grade scalablity and HA with following capabilities:
 
-Azure Application Gateway provides application-level routing and load balancing services for your web front end, with following capabilities:
 * Scalable, highly-available web application delivery
   Azure Application Gateway enable you to build a scalable and highly-available web front end in Azure. You control the size of the gateway and can scale your deployment based on your needs.
 * Efficient, secure web frontend
   SSL offload lets you build a secure web front end with efficient backend servers and also streamline your certificate management.
 * Close integration with Azure services
-  Application Gateway allows easy integration with Azure Traffic Manager to support multi-region redirection, automatic failover, and zero-downtime maintenance. Application Gateway is also integrated with Azure Load Balancer to support scale-out and high-availability for both Internet-facing and internal-only web front ends.
+  Application Gateway allows easy integration with Azure Traffic Manager to support multi-region redirection, automatic failover, and zero-downtime maintenance.
+
+
+## Pre-requisites
 
-# 2 Pre-requisites
+* Setup a Cloud Foundry deployment
+
+* Update the manifest, making sure multiple routers are added and HA Proxy removed.
 
```

Comment:
Update the manifest, making sure multiple routers are added and HA Proxy job removed. For example, remove the `ha_proxy_z1` job.

```
- default_networks:
  - name: cf1
  instances: 1
  name: ha_proxy_z1
  networks:
  ...
```